COPY/usr/lib/systemd/system/evmserverd.service

```diff
@@ -6,6 +6,7 @@ After=syslog.target
 ExecStart=/bin/sh -c "/bin/evmserver.sh start"
 ExecStop=/bin/sh -c "/bin/evmserver.sh stop"
 Type=forking
+Restart=on-failure
```

Comment:
It looks like it can (http://www.freedesktop.org/software/systemd/man/systemd.service.html#RestartForceExitStatus), but I couldn't get it to work using this option.